joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/passes/ast/SimpleAstCreationPassTest.scala

```diff
@@ -1527,19 +1527,15 @@ class SimpleAstCreationPassTest extends AbstractPassTest {
 
     val pushCallReceiverBase = pushCallReceiver.argument(1).asInstanceOf[Identifier]
     pushCallReceiverBase.name shouldBe "_tmp_0"
-    pushCallReceiverBase.argumentIndex shouldBe 1
```

Comment:
has this changed? afaict this is the `_tmp_0` within the `_tmp_0.push` field access, where arg-index 1 seems correct

Reply:
Its already in `pushCallReceiver.argument(1).asInstanceOf[Identifier]`.
I just removed the double testing there.

Reply:
duh, of course